File: Selenium/Activities/Activity2.py
import time
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service = service) as driver:
    driver.get("https://v1.training-support.net/selenium/login-form")
    print("Home Page Title is :",driver.title)
    driver.find_element(By.ID,"username").send_keys("admin")
    driver.find_element(By.ID,"password").send_keys("password")
    #driver.find_element(By.XPATH,"//button[text() = 'Log in']").click()
    login = driver.find_element(By.CSS_SELECTOR, "button:nth-child(4)")
    login.click()
    confirmation_msg = driver.find_element(By.ID,"action-confirmation").text
    print("confirmation_msg is :"+confirmation_msg)

    time.sleep(5)
    driver.close()

File: Selenium/Activities/Activity2.java
package activities;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;
import io.github.bonigarcia.wdm.WebDriverManager;

public class Activity2 {
    public static void main(String[] args) throws InterruptedException {
       // System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/login-form");
        System.out.println("Home Page Title is :" + driver.getTitle());
        driver.findElement(By.id("username")).sendKeys("admin");
        driver.findElement(By.id("password")).sendKeys("password");
        driver.findElement(By.xpath("//button[text() = 'Log in']")).click();
        String comfirmation_msg = driver.findElement(By.id("action-confirmation")).getText();
        System.out.println("comfirmation_msg is :" +comfirmation_msg);

        Thread.sleep(500);
        driver.close();


    }
}
